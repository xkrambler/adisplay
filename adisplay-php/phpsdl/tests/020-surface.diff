002+ Segmentation fault (core dumped)
003+ 
004+ Termsig=11
002- int(150)
003- int(100)
004- int(256)
005- = Create True colors
006- int(150)
007- int(100)
008- NULL
009- = Lock
010- bool(false)
011- int(0)
012- int(0)
013- int(1)
014- NULL
015- int(0)
016- = ColorKey
017- int(0)
018- int(0)
019- int(127)
020- int(0)
021- int(-1)
022- = ColorMod
023- int(0)
024- int(0)
025- int(1)
026- int(2)
027- int(3)
028- = AlphaMod
029- int(0)
030- int(0)
031- int(4)
032- = BlendMode
033- int(0)
034- int(0)
035- bool(true)
036- = ClipRect
037- bool(true)
038- NULL
039- object(SDL_Rect)#%d (4) {
040-   ["x"]=>
041-   int(10)
042-   ["y"]=>
043-   int(20)
044-   ["w"]=>
045-   int(30)
046-   ["h"]=>
047-   int(40)
048- }
049- = Convert
050- bool(true)
051- bool(true)
052- = ConvertPixels
053- int(0)
054- string(0) ""
055- = Free
056- = Done