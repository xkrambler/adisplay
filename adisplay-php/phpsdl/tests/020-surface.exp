= Create Palette
int(150)
int(100)
int(256)
= Create True colors
int(150)
int(100)
NULL
= Lock
bool(false)
int(0)
int(0)
int(1)
NULL
int(0)
= ColorKey
int(0)
int(0)
int(127)
int(0)
int(-1)
= ColorMod
int(0)
int(0)
int(1)
int(2)
int(3)
= AlphaMod
int(0)
int(0)
int(4)
= BlendMode
int(0)
int(0)
bool(true)
= ClipRect
bool(true)
NULL
object(SDL_Rect)#%d (4) {
  ["x"]=>
  int(10)
  ["y"]=>
  int(20)
  ["w"]=>
  int(30)
  ["h"]=>
  int(40)
}
= Convert
bool(true)
bool(true)
= ConvertPixels
int(0)
string(0) ""
= Free
= Done