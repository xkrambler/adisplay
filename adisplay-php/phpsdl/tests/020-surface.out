= Create Palette
Segmentation fault (core dumped)

Termsig=11